web/content/3.0.md

```diff
@@ -36,9 +36,49 @@ plugins. Official documentation [is available here](https://theforeman.org/manua
 
 ### Katello headline features
 
-## Deprecations
+For the full list of changes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-4.2/CHANGELOG.md).
+
+* New Experimental Content View UI Features:
+  * Publish & Promote
+  * Add component CVs to composite CVs
+* Pulp's mirorred metadata is now used to speed up repo syncs
+* Upgrade to Pulp 3.14 (This was backported to Katello 4.1)
+* support deleting RH repos from the repositories details
+
+## Deprecations & Removals
```

Comment:
I think this could be 2 separate sections. For Foreman we have `Upgrade warnings` and `Deprecations`. It'd be nice to be consistent.

Reply:
done!